src/pages/Home/StoreLocator.js

```diff
@@ -23,83 +34,69 @@ const StoreList = styled.div`
-           storeName: "Wish by the Sea",
-           address: "208 1st \nSt. Langley, WA 98260",
-           lat: 48.040886,
-           lng: -122.408415,
-           website: "https://www.facebook.com/wishsisters/",
-           googleMapUrl:
-             "https://www.google.com/maps/place/Wish+by+the+Sea/@48.0408497,-122.4106087,17z/data=!3m1!4b1!4m5!3m4!1s0x548ff9375096a79f:0xd36c343435ee70eb!8m2!3d48.0408497!4d-122.4084147"
-         },
-         {
-           storeName: "Flying Bear Florist",
-           address: "207 1st \nSt. Langley, WA 98260",
-           lat: 48.040654,
-           lng: -122.408515,
-           website: "http://www.flyingbearfarm.com/",
-           googleMapUrl:
-             "https://www.google.com/maps/place/Flying+Bear/@48.0401524,-122.4131517,16z/data=!4m8!1m2!2m1!1sFlying+Bear+Florist!3m4!1s0x548ff9375a98113f:0xfe8fbf5dd6408a40!8m2!3d48.0406478!4d-122.4085255"
-         },
-         {
-           storeName: "Madrona Supply Co.",
-           address: "8754 Washington 525 \nClinton, WA 98236",
-           lat: 47.977944,
-           lng: -122.354667,
-           website: "https://www.madronasupplyco.com/",
-           googleMapUrl:
-             "https://www.google.com/maps/place/Madrona+Supply+Co./@47.9779361,-122.3568657,17z/data=!3m1!4b1!4m5!3m4!1s0x548fff0b130a53fb:0x485e2654adba8037!8m2!3d47.9779361!4d-122.3546717"
-         }
-       ];
+  {
+    storeName: '3 Sisters Market',
+    address: '779 Holbrook Road \nCoupeville, WA 98239',
+    lat: 48.237278,
+    lng: -122.719937,
+    website: 'https://www.3sistersmarket.com/',
+    googleMapUrl:
+      'https://www.google.com/maps/place/3+Sisters+Market/@48.237249,-122.722152,17z/data=!3m1!4b1!4m5!3m4!1s0x548f8c86bc348f65:0x2ea75e0b77df1645!8m2!3d48.237249!4d-122.719958',
+  },
+  {
+    storeName: 'Flying Bear Florist',
+    address: '207 1st \nSt. Langley, WA 98260',
+    lat: 48.040654,
+    lng: -122.408515,
+    website: 'http://www.flyingbearfarm.com/',
+    googleMapUrl:
+      'https://www.google.com/maps/place/Flying+Bear/@48.0401524,-122.4131517,16z/data=!4m8!1m2!2m1!1sFlying+Bear+Florist!3m4!1s0x548ff9375a98113f:0xfe8fbf5dd6408a40!8m2!3d48.0406478!4d-122.4085255',
+  },
+  {
+    storeName: 'Madrona Supply Co.',
+    address: '8754 Washington 525 \nClinton, WA 98236',
+    lat: 47.977944,
+    lng: -122.354667,
+    website: 'https://www.madronasupplyco.com/',
+    googleMapUrl:
+      'https://www.google.com/maps/place/Madrona+Supply+Co./@47.9779361,-122.3568657,17z/data=!3m1!4b1!4m5!3m4!1s0x548fff0b130a53fb:0x485e2654adba8037!8m2!3d47.9779361!4d-122.3546717',
+  },
+];
 
 const StoreLocator = () => {
-
   return (
-    <ComponentWrapper id="findstore" backgroundColor={"#FFF3D1"}>
+    <ComponentWrapper id='findstore' backgroundColor={'#FFF3D1'}>
       <StyledH2>Store Locator</StyledH2>
-      <LocationsWrapper>
-        <GoogleMapWrapper />
-        <StoreList>
-          {stores.map(store => (
+      <LocationsWrapper id='locations-wrapper'>
```

Comment:
Curious on the id's here. What purpose are they serving? I'm not seeing it immediately.

Reply:
I added the ID so I could identify the element in DevTools, I can take it out but I usually add classes or IDs to all my components for that very reason. Is there a reason you didn't add IDs to your project?

Reply:
Generally speaking it's just one more thing to maintain so I opted not to add them. The less complexity the better, especially with class / ID names - typically you assume there's some styling or JS that targets the class/ID. 

That said I *think* there's a way to configure styled components to actually provide the name of the component instead of the random generated gibberish classnames. That's for another day though - for now i'm down with this change, just wanted to make sure I wasn't missing something!